selenium-connect.gemspec

```diff
@@ -14,10 +14,10 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>=1.9'
 
-  s.add_dependency 'selenium-webdriver', '2.53.1'
+  s.add_dependency 'selenium-webdriver'
   s.add_dependency 'rake'
-  s.add_dependency 'sauce'
+  s.add_dependency 'sauce', '~> 3.7'
   s.add_dependency 'sauce-connect'
   s.add_dependency 'sauce_whisk'
-  s.add_dependency 'appium_lib', '~> 6.0.0'
+  s.add_dependency 'appium_lib', '~> 8.0.0'
```

Comment:
Have you tried running the test_selenium_ios tests with appium 8?  If not, I would suggest not upgrading this until it's been tested out.